src/Netflix/Main.java

```diff
@@ -0,0 +1,237 @@
+
+        System.out.println("Welcome to netflix list! Please state your name:");
+        strUser = sc.nextLine();
+        System.out.println("Hi there " + strUser + "! Here are your shows for today.");
+        System.out.println();
+
+        // Main loop
+        while(blnMenu){
+
+            // Displays films
+            newNetflixList.getFilms();
+            System.out.println();
+            System.out.println("What would you like to do?");
+            System.out.println("1. Browse films (Film details)\n2. Customize Lists\n3. Recommendations\n4. Exit");
+            System.out.println();
+            
+            strChoice = sc.nextLine();
+            switch(strChoice){
+                case "1":
+                    System.out.println("Which film would you like to browse? (Type the number of the film): ");
+                    intFilm = sc.nextInt();
+                    sc.nextLine();
+                    System.out.println();
+
+                    // Gets movies and shows to display
+                    if(intFilm < 6){
+                        System.out.println(newNetflixList.getMovie(intFilm - 1));
+                    }
+                    else if(intFilm > 5){
+                        System.out.println(newNetflixList.getShow(intFilm - 6));
+                    }
+
+                    // Uses setter method to set film as "favourite"
+                    System.out.println("Type 'favourite' if you would like to favourite this film.\nType anything else to continue");
+                    if(sc.nextLine().equalsIgnoreCase("favourite")){
+                        if(intFilm < 6 ){
+                            newNetflixList.getMovie(intFilm - 1).setFavourite();
+                        }
+                        else if(intFilm > 5 && intFilm < 11){
+                            newNetflixList.getShow(intFilm - 6).setFavourite();
+                        }
+                    }
+                    break;
+
+                case "2":
+                    System.out.println();
+                    System.out.println("Here is your list so far: ");
+
+                    // Displays all films in the user's list
+                    for(int i = 0; i < 10; i++){
+                        if((i + 1) <= filmList.size()){
+                            System.out.println("[" + filmList.get(i) + "]");
+                        }
+                        else{
+                            System.out.println("[EMPTY]");
+                        }
+                    }
+
+                    System.out.println("\nType any key to continue...");
+                    sc.nextLine();
+                    System.out.println("Would you like to add to your list? Yes/No");
+
+                    // Makes sure that second loop doesnt automatically open the list
+                    blnList = false;
+                    strChoice = sc.nextLine();
+
+                    if(strChoice.equalsIgnoreCase("yes") && filmList.size() <= 10){
```

Comment:
the hardcoding of the value `10` in this case goes against encapsulation principles.  This should be at least defined with a variable.  The better option would be to have a UserFilmList class to manage the user list.